clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/model/GoogleServerGroup.groovy

```diff
@@ -67,6 +68,8 @@ class GoogleServerGroup {
   @Canonical
   class View implements ServerGroup {
     final String type = GoogleCloudProvider.GCE
+    static final String REGIONAL_LB_NAMES = "regionalLoadBalancerNames"
```

Comment:
Is it worth it to abbreviate with `LB` here? We very rarely abbreviate these names, and we use `loadBalancer` all over the place.